nativelib/src/main/scala/java/lang/Class.scala

```diff
@@ -15,10 +15,21 @@ final class _Class[A](val ty: Ptr[Type]) {
       false
   }
 
+  def getComponentType(): _Class[_] = {
+    if (ty == typeof[BooleanArray]) classOf[scala.Boolean]
+    else if (ty == typeof[CharArray]) classOf[scala.Char]
+    else if (ty == typeof[ByteArray]) classOf[scala.Byte]
+    else if (ty == typeof[ShortArray]) classOf[scala.Short]
+    else if (ty == typeof[IntArray]) classOf[scala.Int]
+    else if (ty == typeof[LongArray]) classOf[scala.Long]
+    else if (ty == typeof[FloatArray]) classOf[scala.Float]
+    else if (ty == typeof[DoubleArray]) classOf[scala.Double]
+    else classOf[java.lang.Object]
```

Comment:
It is indeed an approximation for the time being. The goal here was to unblock reverse. There is a separate ticket for precise semantics.